Documentation/operations/performance/index.rst

```diff
@@ -60,8 +60,7 @@ the Packet machines to use a `"Mixed/Hybrid"
 <https://www.packet.com/developers/docs/network/advanced/layer-2/>`_ network
 mode, where the secondary interfaces of the machines share a flat L2 network.
 While this can be done on the Packet web UI, we include appropriate Terraform
-files to automate this process.
-
+(version 0.13) files to automate this process.
```

Comment:
Should this be `>=0.13`?

Reply:
I don't know how they manager their versions or what expectations we should have (especially wrt the packet plugin). It's probably OK, but I thought it's better not to take the risk. I'd be happy to change it.

Reply:
Hm, it's still <1.x so likely to break API maybe? So yeah, best to keep it as is I guess.